src/components/Header/AppsMenu.js

```diff
@@ -28,6 +29,17 @@ const AppsMenu = () => {
         Studies and Access
       </a>
     </Menu.Item>,
+    <Menu.Item key="kf-forum">
+      <a href={`${'todo'}/todo`} target={'_blank'} rel={'noopener noreferrer'}>
+        <MessageOutlined style={{ fontSize: '15px', marginRight: 0 }} />
```

Comment:
it seems that we're forced to use inline-styling

Reply:
why? https://ant.design/components/icon/#API has an api for className.